docs/design/autoware-interfaces/index.md

```diff
@@ -0,0 +1,5 @@
+# Autoware Interfaces
```

Comment:
Please use `sentence case`.
https://developers.google.com/style/headings?hl=en#heading-and-title-format

```suggestion
# Autoware interfaces
```